src/components/UI/ApplyButton.js

```diff
@@ -0,0 +1,13 @@
+const ApplyButton = () => {
```

Comment:
Great work on making a button functionality.
To make this button truly reusable, you can add the button text as a prop and give it here.
This will allow this component to show anything we want and not only "Apply".

You can also add a snapshot test for this button component.

This is only a suggestion if you want to use the button in other areas of the website.

Reply:
Great reviewsss !!! thank u

Reply:
You are most welcome :)